client/extensions/wp-job-manager/components/settings/job-submission/index.jsx

```diff
@@ -1,5 +1,221 @@
+import ReduxFormSelect from 'components/redux-forms/redux-form-select';
+import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
+import ReduxFormToggle from 'components/redux-forms/redux-form-toggle';
+import SectionHeader from 'components/section-header';
+
+const JobSubmission = ( { isDisabled, submissionDuration, translate } ) => {
+	return (
+		<div>
+			<form>
+				<FormSection name="account">
+					<SectionHeader label={ translate( 'Account' ) }>
+						<FormButton compact
+							disabled={ isDisabled } />
+					</SectionHeader>
+					<Card>
+						<FormFieldset>
+							<ReduxFormToggle
+								disabled={ isDisabled }
+								name="isRequired"
+								text={ translate( 'Require an account to submit listings' ) } />
+							<FormSettingExplanation isIndented>
+								{ translate( 'Limits job listing submissions to registered, logged-in users.' ) }
+							</FormSettingExplanation>
+
+							<ReduxFormToggle
+								disabled={ isDisabled }
+								name="enableRegistration"
+								text={ translate( 'Enable account creation during submission' ) } />
+							<FormSettingExplanation isIndented>
+								{ translate( 'Includes account creation on the listing submission form, to allow ' +
+									'non-registered users to create an account and submit a job listing simultaneously.' ) }
+							</FormSettingExplanation>
+
+							<ReduxFormToggle
+								disabled={ isDisabled }
+								name="generateUsername"
+								text="Generate usernames from email addresses" />
+							<FormSettingExplanation isIndented>
+								{ translate( 'Automatically generates usernames for new accounts from the registrant\'s ' +
+									'email address. If this is not enabled, a "username" field will display instead.' ) }
+							</FormSettingExplanation>
+						</FormFieldset>
+
+						<FormFieldset>
+							<FormLabel>
+								{ translate( 'Role' ) }
+							</FormLabel>
+							<ReduxFormSelect
+								disabled={ isDisabled }
+								name="role">
+								<option value="editor">{ translate( 'Editor' ) }</option>
+								<option value="author">{ translate( 'Author' ) }</option>
+								<option value="contributor">{ translate( 'Contributor' ) }</option>
+								<option value="subscriber">{ translate( 'Subscriber' ) }</option>
+								<option value="teacher">{ translate( 'Teacher' ) }</option>
+								<option value="employer">{ translate( 'Employer' ) }</option>
+								<option value="customer">{ translate( 'Customer' ) }</option>
+								<option value="shop_manager">{ translate( 'Shop manager' ) }</option>
+							</ReduxFormSelect>
+							<FormSettingExplanation>
+								{ translate( 'Any new accounts created during submission will have this role.' ) }
+							</FormSettingExplanation>
+						</FormFieldset>
+					</Card>
+				</FormSection>
+			</form>
+
+			<form>
+				<FormSection name="approval">
+					<SectionHeader label={ translate( 'Approval' ) }>
```

Comment:
Hi! I've found a possible matching string that has already been translated **3** times:
`translate( 'Waiting approval' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/wpjm-job-submission)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).